apps/frontend/src/store/evaluations.ts

```diff
@@ -35,6 +37,38 @@ export class Evaluation extends VuexModule {
       });
   }
 
+  @Action
+  findEvaluationsByIds(evaluationIds: string[]): IEvaluation[] {
+    return this.allEvaluations.filter((evaluation) =>
+      evaluationIds.includes(evaluation.id)
+    );
+  }
+
+  @Action
+  async load_results(evaluations: IEvaluation[]): Promise<(FileID | void)[]> {
```

Comment:
If the type here is loosened down from a whole `IEvaluation[]` down to `{id: string}[]` then it helps in some other areas.